include/nanogui/button.h

```diff
@@ -90,6 +90,8 @@ class NANOGUI_EXPORT Button : public Widget {
     std::function<void()> mCallback;
     std::function<void(bool)> mChangeCallback;
     std::vector<Button *> mButtonGroup;
+public:
+	EIGEN_MAKE_ALIGNED_OPERATOR_NEW
```

Comment:
Regarding all the ``EIGEN_MAKE_ALIGNED_OPERATOR_NEW`` changes, I am wondering: could we instead use the new c++11 alignas keyword? This would involve writing ``class NANOGUI_EXPORT alignas(16) Button``. In this case, it would be nicer to replace ``NANOGUI_EXPORT`` with another macro ``NANOGUI_EXPORT_CLASS`` that incluces the keyword.